src/main/java/com/checkout/disputes/DisputesClientImpl.java

```diff
@@ -12,15 +12,19 @@
 
 public class DisputesClientImpl extends AbstractClient implements DisputesClient {
 
-    private static final String DISPUTES = "/disputes";
-    private static final String FILES = "/files";
+    private static final String DISPUTES = "disputes";
+    private static final String FILES = "files";
     private static final String ACCEPT = "accept";
     private static final String EVIDENCE = "evidence";
 
     public DisputesClientImpl(final ApiClient apiClient, final CheckoutConfiguration configuration) {
         super(apiClient, configuration, SdkAuthorizationType.SECRET_KEY);
     }
 
+    protected DisputesClientImpl(final ApiClient apiClient, final CheckoutConfiguration configuration, final SdkAuthorizationType sdkAuthorizationType) {
```

Comment:
So, basically the extra constructor is to set a different `SdkAuthorizationType`?